cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala

```diff
@@ -325,6 +325,31 @@ class CallSubqueryTest extends DocumentingTest {
       p("""As the size of the CSV file in this example is small, only a single separate transaction is
           #started and committed.""".stripMargin('#'))
       note(p("""`CALL { ... } IN TRANSACTIONS` is only allowed in <<query-transactions, implicit transactions>>."""))
+      section("Deleting a large amount of nodes", "delete-with-call-in-transactions") {
+        p("""Using `CALL {} IN TRANSACTIONS` is the recommended way of deleting a large amount nodes.""".stripMargin)
+        addQuery(
+          """MATCH (n)
+            #CALL {
+            #  WITH n
+            #  DETACH DELETE n
+            #} IN TRANSACTIONS""".stripMargin('#'),
+          assertions = ResultAssertions(r => r.isEmpty),
+          databaseStateBehavior = ClearState,
+        ) { resultTable() }
+        important(p(
+          """The `CALL { } IN TRANSACTIONS` subquery above has a special treatment in the database to ensure optimal performance.
+            |Modifying the clause may result in `OutOfMemory` exceptions for sufficiently large datasets.""".stripMargin))
```

Comment:
I'm not sure I understand the use of "special treatment" this note. Perhaps something like this as an alternative?

```suggestion
          """The `CALL { } IN TRANSACTIONS` subquery is handled by the database so as to ensure optimal performance.
            |Modifying the clause may result in `OutOfMemory` exceptions for sufficiently large datasets.""".stripMargin))
```